src/ui/atoms/theme/ThemeSwitcher.tsx

```diff
@@ -50,15 +48,17 @@ export const ThemeSwitcher: React.FC<ThemeSwitcherProps> = ({
     setTheme(value as Theme)
   }, [prefersColorDark, setTheme, value])
 
-  const switchIcon = isDarkTheme ? <SunIcon /> : <MoonIcon />
-
   return (
     <Switch.Root
       checked={theme === 'dark'}
       className="okp4-theme-switcher-main"
       onCheckedChange={handleCheck}
     >
-      <div className={`okp4-theme-switcher-icon ${theme}`}>{switchIcon}</div>
+      <div className={`okp4-theme-switcher-icon ${theme}`}>
+        <svg height={18} width={18}>
```

Comment:
Why don't you simply use the Icon component here?

Reply:
This is a work around to be able to have the moon in white. The `icon component` would render a blue moon and we have not found a way to override the icon colours as of now.